nilearn/plotting/displays.py

```diff
@@ -946,6 +1024,138 @@ def draw_cross(self, cut_coords=None, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def add_graph(self, adjacency_matrix, node_coords,
+                  node_color=None, node_size=None,
+                  edge_cmap=None, edge_threshold=None,
+                  edge_kwargs=None, node_kwargs=None):
+        """Plot undirected graph on each of the axes
+
+            Parameters
+            ----------
+            adjacency_matrix: numpy array of shape (n, n)
+                represents the edges strengths of the graph. Assumed to be
+                a symmetric matrix.
+            node_coords: numpy array_like of shape (n, 3)
+                3d coordinates of the graph nodes in world space.
+            node_color: color or sequence of colors
+                color(s) of the nodes.
+            node_size: scalar or array_like
+                size(s) of the nodes in points^2.
+            edge_cmap: colormap
+                colormap used for representing the strength of the edges.
+            edge_threshold: str or number
+                If it is a number only the edges with a value greater than
+                edge_threshold will be shown.
+                If it is a string it must finish with a percent sign,
+                e.g. "25.3%", and only the edges with a abs(value) above
```

Comment:
> Any further thoughts or objections?

I think that I would like the percentage to be the filling factor of the
graph. If I say threshold="80%", I want to see ".2 * n (n - 1) / 2"
edges.

That way, we don't have an unstable behavior depending on wether people
threshold the matrices or not.